docs/configuration.md

```diff
@@ -1207,6 +1207,25 @@ Apart from these, the following properties are also available, and may be useful
 <tr>
   <td><code>spark.ui.view.acls</code></td>
   <td>Empty</td>
+</tr>
+<tr>
+  <td>spark.executor.multiPerWorker</td>
+  <td>false</td>
+  <td>
+    enable user to run multiple executors in the same worker.
+  </td>
+</tr>
+<tr>
+  <td>spark.executor.maxCoreNumPerExecutor</td>
```

Comment:
I agree with @nishkamravi2's suggestion to just reuse `spark.executor.cores` here. No reason to introduce another hard-to-name config that we can't remove in the future.